crates/bevy_ui/src/widget/text.rs

```diff
@@ -70,44 +166,29 @@ pub fn text_system(
 
     let scale_factor = ui_scale.scale * window_scale_factor;
 
-    let inv_scale_factor = 1. / scale_factor;
-
     #[allow(clippy::float_cmp)]
     if *last_scale_factor == scale_factor {
```

Comment:
Why do we check for a changed scale twice?
Perhaps we can simplify this logic in a future PR

Reply:
This might be a bug even maybe it could result in extra text recomputations, I'm not sure. Needs testing.

Reply:
I think it's redundant but it shouldn't result in extra recomputations at least. I want to leave it as I think the scale factor is so complicated and super fragile it needs to be sorted in its own PR, like you say.